### PR TITLE
Feature/terra3 105 more accounts for ecr repo access

### PR DESCRIPTION
**Pull request by Kaushikkatariya**
Please review my PR.

### CODE REVIEW

File: variables.tf
Comment (w13k) [inline variables.tf:1]: Please add descriptions for both.

File: modules/ecr/main.tf [hunk 1]
```diff
@@ -12,25 +12,42 @@ resource "aws_ecr_repository" "ecr_repo" {
   }
 }
 
+# Create the list of account ids, works for both the parameters => ID and IDs.
+locals {
+  aws_principals_account_id_list = length(var.access_for_account_id) > 0 ? [
```
Comment (w13k) [inline modules/ecr/main.tf]: A more clean approach is to just work here with the account_ids and do the arn formatting in line 44 one time.

**Review by w13k — CHANGES_REQUESTED** [review]
Please check out the two comments and merge by yourself! Other than that it LGTM.

File: variables.tf
Comment (Kaushikkatariya) [reply variables.tf:1]: The description has been added to both the variables.

File: modules/ecr/main.tf (hunk 1 above)
Comment (Kaushikkatariya) [reply modules/ecr/main.tf]: The clean approach has been performed.

**Review by w13k — APPROVED** [review]

**Comment by w13k** [comment]
This PR is included in version 1.21.0 :tada:

**Pull request merged**